extensions/wikia/WeiboTag/WeiboTagController.class.php

```diff
@@ -0,0 +1,76 @@
+<?php
+class WeiboTagController extends WikiaParserTagController {
+	const PARSER_TAG_NAME = 'weibo';
+	const TAG_SRC = 'http://widget.weibo.com/relationship/bulkfollow.php?';
+	const TAG_SOURCE_ALLOWED_PARAMS_WITH_DEFAULTS = [
+		'color' => '',
```

Comment:
Weibo's default is `C2D9F2,FFFFFF,0082CB,666666`. Does setting it to an empty string overrides them? The same goes for all params. See http://jssdk.sinaapp.com/widget/bulkfollow.php#wb for partial list of defaults.